Wangscape/tilegen/TilesetGenerator.cpp

```diff
@@ -30,7 +28,8 @@ void TilesetGenerator::generate(std::function<void(const sf::Texture&, std::stri
     boost::filesystem::path p(options.relativeOutputDirectory);
     for (const auto& clique : options.cliques)
     {
-        std::pair<size_t, size_t> tileset_resolution = calculateTilesetResolution(clique.size());
+        CornersGenerator cp(clique, static_cast<size_t>(CORNERS));
```

Comment:
Find a move descriptive name for it, please. `cornerGenerator` is good enough. You don't want to wonder, what does `cp` mean e.g. next month :wink:.